bigquery-antipattern-recognition/cicd/README.md

```diff
@@ -0,0 +1,69 @@
+# Using BigQuery AntiPattern Tool in CI/CD
```

Comment:
can we mice the cicd folder into a subfolder of the root folder. the current folder is for the java code